doc/user_guide/user_guide_107.md

```diff
@@ -0,0 +1,143 @@
+# Exasol Power BI Connector 1.0.7 Release - User Guide
+
+## New features
+
+- Custom Query support
+
+- ODBC Driver Logging
+
+- DSN (Data Source Name) Support
+
+- Limited Kerberos support
+
+## Installing the connector:
+
+### Download the .mez
+
+Copy the ''Exasol.mez' file available on : [Releases · exasol/powerbi-exasol · GitHub](https://github.com/exasol/powerbi-exasol/releases)
+
+You can find it separately under the assets dropdown.
+
+This is the direct link:
+
+https://github.com/exasol/powerbi-exasol/releases/download/1.0.7/Exasol.mez
+
+### Copy the .mez to the following directory
+
+Copy the mez file to `C:\Users\\<YourAccount>\Documents\Microsoft Power BI Desktop\Custom Connectors` Directory. 
+
+Create the directory path if it doesn't exist.
+
+### Change the PowerBI Desktop settings
+
+Go to options in PowerBI:
+
+File > Options and settings > Options
+
+![](./img/2021-07-16-13-42-57-image.png)
```

Comment:
:+1 nice to have images here

Reply:
Could you rename them, with speaking names.

Reasons:
* Helps with screen readers
* Helps for search engine optimization